### PR TITLE
DB_Seeding-update agreement assigned offer types

### PR DESCRIPTION
**Pull request by jjeroch**

### CODE REVIEW

**Comment by jjeroch** [comment]
@evegufy & @Phil91  I changed the file name and updated the content. Not a urgent PR

File: src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreement_assigned_offer_types.json [hunk 1]
```diff
@@ -10,5 +10,9 @@
   {
     "agreement_id": "aa0a0000-7fbc-1f2f-817f-bce0502c1016",
     "offer_type_id": 1
+  },
+  {
+    "agreement_id": "aa0a0000-7fbc-1f2f-817f-bce0502c1092",
```
Comment (Phil91) [inline src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreement_assigned_offer_types.json:15]: The agreementId isn't yet existing in the files, therefore the migratory currently runs into an error

**Review by Phil91 — CHANGES_REQUESTED** [review]

File: src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreement_assigned_offer_types.json (hunk 1 above)
Comment (jjeroch) [reply src/portalbackend/PortalBackend.Migrations/Seeder/Data/agreement_assigned_offer_types.json:15]: fixed; it must be 91 instead of 92

**Comment by jjeroch** [comment]
@Phil91  can you help me out with the branch conflict?

**Comment by Phil91** [comment]
> @Phil91 can you help me out with the branch conflict?

Sure, I'll have a look at it and test it again

**Review by Phil91 — APPROVED** [review]

**Pull request merged**